base/grafana/grafana.Deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         app: grafana
     spec:
       containers:
-      - image: index.docker.io/sourcegraph/grafana:6.3.3-2@sha256:f84e4260dce4e36f5d5e9c6deb393ac1e9c930ffae622c432fa86640a1f2699f
+      - image: index.docker.io/sourcegraph/grafana:6.3.3@sha256:2f68b9b1542e7d75459d983b606d2fdd1c11a75610464e3d7a6ced4f3ac474bf
```

Comment:
@uwedeportivo I don't know why Renovate would make this change? that's odd

Reply:
I think it's because in semver `MAJOR.minor.patch-prerelease` < `MAJOR.minor.patch`.

Since Renovate updated this PR to include all the tag updates to `3.8.0-rc.2`, I've reverted the Grafana tag update in this PR before merging it. Maybe we need a `sourcegraph/grafana:6.3.4`?